core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/popup/info/UIViewMetadataForm.java

```diff
@@ -173,6 +173,19 @@ public void execute(Event<UIViewMetadataForm> event) throws Exception {
               }
               if(!node.hasProperty(name) || (node.hasProperty(name) && !node.getProperty(name).getString().equals(value)))
                 node.setProperty(name, value);
+            } else if (requiredType == 4){ // double
+              UIFormInput uiInput = uiForm.getUIInput(inputName);
+              double value = 0;
+              if(uiInput != null && StringUtils.isNotBlank((String)uiInput.getValue())) {
+                if ((!node.hasProperty(name) || (node.hasProperty(name) && node.getProperty(name).getDouble() != 0))) {
+                  try {
+                    value =  Double.parseDouble((String) uiInput.getValue());
+                    node.setProperty(name, value);
+                  } catch (Exception e) {

```

Comment:
You should catch only the exception "NumberFormatException" and you shouldn't delete the value, but you should discard changes and inform the user that the number format is wrong for this field. So add a LOG.error + throw a new Runtime Exception with an internationalized message for this.